app/javascript/controllers/conditional_question_controller.js

```diff
@@ -28,7 +28,10 @@ export default class extends Controller {
           div.style.display = "block"
         } else {
           div.style.display = "none"
-          let buttons = document.getElementsByName(`case_log[${targetQuestion}]`)
+          let buttons = document.getElementsByName(`case_log[${targetQuestion}]`);
+          if (buttons.length == 0){
```

Comment:
What case does this happen in?

Reply:
Ah I see, so we probably need case handling for all the different target types (in the long run). Might be worth adding a comment or a TODO there that that'll currently only clear the answer for radio and checkbox, and probably won't work yet for other fields?

Reply:
Interesting. Well :+1: then.